Sources/DataPipeline/CustomerIO+Events.swift

```diff
@@ -23,7 +23,7 @@ public extension CustomerIO {
     ///      https://segment.io/libraries/ios#ids
     /// - traits: A dictionary of traits you know about the user. Things like: email, name, plan, etc.
     func identify<T: Codable>(userId: String, traits: T?) {
-        CIODataPipeline.analytics.identify(userId: userId, traits: traits)
+        DataPipeline.shared.identify(identifier: userId, body: traits)
```

Comment:
I just changed one call in this PR to demonstrate, future PRs will remove `CIODataPipeline` completely and all calls will look like this